source/_docs/installation/raspberry-pi.markdown

```diff
@@ -41,7 +41,7 @@ sudo apt-get upgrade -y
 Install the dependencies.
 
 ```bash
-sudo apt-get install python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev autoconf build-essential libopenjp2-7
+sudo apt-get install python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libtiff-dev autoconf build-essential libopenjp2-7
```

Comment:
Is `libtiff-dev` needed in general or only if you want to use certain integrations?

Reply:
It is needed in general nowadays and a requirement to run Home Assistant.